sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeClientOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Storage.Blobs;
+using Azure.Storage.Files.DataLake.Models;
+
+namespace Azure.Storage.Files.DataLake
+{
+#pragma warning disable AZC0008 // ClientOptions should have a nested enum called ServiceVersion
```

Comment:
This one can't be skipped.  Every ClientOptions needs to have a ServiceVersion enum and take it in the .ctor following the pattern.

Reply:
Why?  DataLakeClientOptions extends BlobClientOptions, so it has the ServiceVersion enum and exposes it's parent's constructors.

Reply:
I derived from BlobClientOptions because I don't see a reason why DataLakeClientOptions shouldn't be identical to BlobClientOptions other than the name  of the class, and the namespace.  Why write extra code for the same functionality?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/8394